src/platforms/common/cdcacm.c

```diff
@@ -188,14 +188,14 @@ static const struct usb_endpoint_descriptor uart_data_endp[] = {{
 	.bDescriptorType = USB_DT_ENDPOINT,
 	.bEndpointAddress = 0x03,
 	.bmAttributes = USB_ENDPOINT_ATTR_BULK,
-	.wMaxPacketSize = CDCACM_PACKET_SIZE,
+	.wMaxPacketSize = 48, //CDCACM_PACKET_SIZE,
```

Comment:
It does indeed need to change, which is why I deliberately left this grot in so it would be seen.  If we change the size of CDCACM_PACKET_SIZE it will affect all the CDCs, including the gdb one which _does_ deal with fast data and probably benefits from the large size buffer. I worked out that the overall buffer was 32 octets overcommitted, and indeed the second 32 bytes of the trace buffer were corrupt. a new definition for the second serial port size is probably needed, assuming your assessment of USB allocation is the same as mine.